packages/params/src/forkName.ts

```diff
@@ -20,28 +20,70 @@ export enum ForkSeq {
   deneb = 4,
 }
 
+function exclude<T>(coll: T[], val: T[]): T[] {
+  return coll.filter((f) => !val.includes(f));
+}
+
+export function highestFork<F extends ForkName>(forkNames: F[]): F {
+  let highest = forkNames[0];
+
+  for (const forkName of forkNames) {
+    if (ForkSeq[forkName] > ForkSeq[highest]) {
+      highest = forkName;
+    }
+  }
+
+  return highest;
+}
+
+export function lowestFork<F extends ForkName>(forkNames: F[]): F {
+  let lowest = forkNames[0];
+
+  for (const forkName of forkNames) {
+    if (ForkSeq[forkName] < ForkSeq[lowest]) {
+      lowest = forkName;
+    }
+  }
+
+  return lowest;
+}
+
 export type ForkAll = ForkName;
+export const forkAll = Object.keys(ForkName) as ForkName[];
```

Comment:
I think this should rather iterate over the values, it works both ways as keys equal values but it also removes the need to have type cast

```suggestion
export const forkAll = Object.values(ForkName);
```